src/App.jsx

```diff
@@ -1,35 +1,145 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from 'react';
+import './App.css';
+import Radio from './Components/Radio';
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [value, setValue] = useState(''); 
+  const [value1, setValue1] = useState('1');
+  const [value2, setValue2] = useState('1');
+  const [value3, setValue3] = useState('1');
+  const [value4, setValue4] = useState('1');
+  const [value5, setValue5] = useState('1');
+  const [value6, setValue6] = useState('1');
+  const [value7, setValue7] = useState('1');
+  const [selected, setSelected] = useState(false);
+
+  const toogleDisable = () => {
+    alert('Toggle disabled');
+  }
+
+  const onChange = (e) => {
+    setValue(e.target.value);
+  }
+
+  const onChangeVerticalRadio = (e) => {
+    setValue1(e.target.value);
+    if (e.target.value === '4') {
+      setSelected(true);
+    } else {
+      setSelected(false);
+    }
+  }
+
+  const handleblockDiv1 = (e) => {
+    setValue2(e.target.value);
+  }
+
+  const handleblockDiv2 = (e) => {
+    setValue3(e.target.value);
+  }
+
+  const handleblockDiv3 = (e) => {
+    setValue4(e.target.value);
+  }
+
+  const handleRadioStyle1 = (e) => {
+    setValue5(e.target.value);
+  }
+
+  const handleRadioStyle2 = (e) => {
+    setValue6(e.target.value);
+  }
+
+  const handleRadioStyle3 = (e) => {
+    setValue7(e.target.value);
+  }
 
   return (
     <>
-      <div>
-        <a href="https://vite.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
```

Comment:
avoid inline function in JSX. & make Reuse Button Component.

Reply:
ok I will